src/javascript.rs

```diff
@@ -121,6 +126,39 @@ impl<'a> Generator<'a> {
         }
     }
 
+    fn import(
+        &mut self,
+        module: &'a Vec<String>,
+        as_name: &'a Option<String>,
+        unqualified: &'a Vec<UnqualifiedImport>,
+    ) -> Document<'a> {
+        let module_name = as_name
+            .as_ref()
+            .map(|n| n.as_str().to_doc())
+            .unwrap_or_else(|| Document::String(module.join("_")));
```

Comment:
I think we should only use the last section, as we do in Gleam. If we use the full path it is possible to have a collision between two modules like so:

```gleam
import one/two
import one_two
```
```js
import * as one_two from "./one/two";
import * as one_two from "./one_two";
```

Reply:
Agreed and Fixed